src/components/Grid/Column.js

```diff
@@ -1,46 +1,64 @@
-import PropTypes from "prop-types";
 import styled from "styled-components";
-import constants from "../../theme/constants";
+import { mediumAndUp, largeAndUp, xLargeAndUp } from "../../theme/mediaQueries";
 import spacing from "../../theme/spacing";
 
 const MAX_COLUMNS = 12;
 
 const getSize = val => val / MAX_COLUMNS * 100;
 
-const getPadding = size => props => {
-  if ((props[size] || props.large || props.medium || props.small) === 12)
-    return 0;
+const getFlexProps = val =>
+  val === 12
+    ? `
+      flex-grow: 0;
```

Comment:
Can we use flex instead which is the shorthand for flex-grow, flex-shrink and flex-basis combined.